tests/python/unittest/test_target_codegen_llvm.py

```diff
@@ -784,26 +784,37 @@ def dotest(do_vectorize):
     dotest(True)
     dotest(False)
 
+def test_llvm_crt_static_lib():
+    A = te.placeholder((32, ), dtype='bfloat16')
+    B = te.placeholder((32, ), dtype='bfloat16')
+    d = te.compute((32, ), lambda x: A[x] + B[x])
+    sch = te.create_schedule(d.op)
+    module = tvm.build(sch, [A, B, d], target=tvm.target.create('llvm --system-lib --runtime=c'))
+    print(module.get_source())
+    module.save('test.o')
+
+
 if __name__ == "__main__":
-    test_multiple_func()
-    test_llvm_large_uintimm()
-    test_llvm_import()
-    test_alignment()
-    test_rank_zero()
-    test_rank_zero_bound_checkers()
-    test_llvm_bool()
-    test_llvm_persist_parallel()
-    test_llvm_condition()
-    test_llvm_vadd_pipeline()
-    test_llvm_add_pipeline()
-    test_llvm_intrin()
-    test_llvm_overloaded_intrin()
-    test_llvm_flip_pipeline()
-    test_llvm_madd_pipeline()
-    test_llvm_temp_space()
-    test_llvm_lookup_intrin()
-    test_llvm_div()
-    test_llvm_fp_math()
-    test_dwarf_debug_information()
-    test_llvm_shuffle()
-    test_llvm_bf16()
+    # test_multiple_func()
```

Comment:
did you disable these unintentionally?

Reply:
whoops, after the flaky CI issue is resolved i'll push a patch with this reverted

Reply:
uncommented